src/implementations/twig/components/expandable/expandable.story.js

```diff
@@ -1,47 +1,65 @@
-import { withKnobs, text, select, optionsKnob } from '@storybook/addon-knobs';
 import { withNotes } from '@ecl/storybook-addon-notes';
-import { getExtraKnobs, tabLabels } from '@ecl/story-utils';
 import withCode from '@ecl/storybook-addon-code';
-import he from 'he';
+import { correctSvgPath } from '@ecl/story-utils';
 
-import defaultSprite from '@ecl/resources-ec-icons/dist/sprites/icons.svg';
 import demoData from '@ecl/specs-component-expandable/demo/data';
-
 import expandable from './expandable.html.twig';
 import notes from './README.md';
 
-const prepareExpandable = (data) => {
-  data.id = select('id', [data.id], data.id, tabLabels.required);
-  data.label_expanded = text(
-    'label_expanded',
-    data.label_expanded,
-    tabLabels.required
-  );
-  data.label_collapsed = text(
-    'label_collapsed',
-    data.label_collapsed,
-    tabLabels.required
-  );
-  data.content = he.decode(text('content', data.content, tabLabels.required));
-  data.button.icon.path = optionsKnob(
-    'button.icon.path',
-    { current: defaultSprite, 'no path': '' },
-    defaultSprite,
-    { display: 'inline-radio' },
-    tabLabels.required
-  );
-
-  getExtraKnobs(data);
+const getArgTypes = (data) => {
+  return {
+    label_collapsed: {
+      name: 'label of the button',
+      type: { name: 'string', required: true },
+      defaultValue: data.label_collapsed,
+      description: 'Used when the content is hidden.',
```

Comment:
as we have never used ending lines dots, it may be better to continue on this basis?